CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.6)
+
+if (WIN32 AND DEFINED ENV{VCPKG_ROOT})
+  set(CMAKE_TOOLCHAIN_FILE "$ENV{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake")
+endif ()
+
+set(n clinfo)
```

Comment:
I would not be so lazy and rather use `${CMAKE_PROJECT_NAME}` when referring later to the name. At least make it something memorable, like `${Name}`. (I tend to use different capitalization for temporary vars.)

Reply:
You mean `n`? I like it as it is, and it's nothing to do with laziness. You can suggest the change you suggest if the owner accepts.

Reply:
I agree with @ProGTX. Just writing `clinfo` everywhere is much clearer than a cryptic `${n}`